### ISSUE TITLE
Add auto-purge feature

### ISSUE THREAD

**Issue by ceribik** [stephanmahieu/formhistorycontrol-2#2]
Can you add a feature that automatically (every day) purges entries stored > x days??

**Comment by stephanmahieu** [comment]
Yes, an auto-cleanup feature is on my todo list, as well as a whitelist/blacklist feature.
